05_syntax_lab/05.py

```diff
@@ -1,5 +1,9 @@
-"""
-Write a program that randomizes integers in a loop
-until it finds a number that is divisable by: 7, 13 and 15
-"""
-
+from random import randint
+
+while True:
+    num = randint(1,1000000)
+    if num % 7 == 0:
+        if num % 13 == 0:
+            if num % 15 == 0:
+                 print num
+                 break;

```

Comment:
This could be better written in a single if statement connection the conditions with 'and'.